xmeffects.js

```diff
@@ -1,8 +1,14 @@
+(function (window) {
+if (!window.XMPlayer) {
+  window.XMPlayer = {};
+}
+var player = window.XMPlayer;
```

Comment:
Why does this work? If xmeffects.js loads before xm.js, won't `var player` here be divorced from the actual XMPlayer in xm.js?